docs/cue/reference/components.cue

```diff
@@ -935,42 +935,67 @@ components: {
 		}
 
 		env_vars: {
-			HTTP_PROXY: {
+			_http_proxy: {
 				description: """
 					The global URL to proxy HTTP requests through.
 					If another HTTP proxy is set in the configuration file or at a component level,
 					this one will be overriden.
+					The lowercase variant has priority over the uppercase one.
 					"""
 				type: string: {
 					default: null
 					examples: ["http://foo.bar:3128"]
 					syntax: "literal"
 				}
 			}
-			HTTPS_PROXY: {
+			_https_proxy: {
 				description: """
 					The global URL to proxy HTTPS requests through.
 					If another HTTPS proxy is set in the configuration file or at a component level,
 					this one will be overriden.
+					The lowercase variant has priority over the uppercase one.
 					"""
 				type: string: {
 					default: null
 					examples: ["http://foo.bar:3128"]
 					syntax: "literal"
 				}
 			}
-			NO_PROXY: {
+			_no_proxy: {
 				description: """
-						List of hosts to avoid proxying globally.
-						If another no_proxy value is set in the configuration file or at a component level,
-						this one will be overriden.
-					"""
+					List of hosts to avoid proxying globally.
+					If another no_proxy value is set in the configuration file or at a component level,
+					this one will be overriden.
+					The lowercase variant has priority over the uppercase one.
```

Comment:
```suggestion
					List of hosts to avoid proxying globally.
					
					Allowed patterns here include:

                     - Domain names. For example, `example.com` will match requests to to `example.com`
                     - Wildcard domains. For example, `.example.com` will match requests to `example.com` and its subdomains
                     - IP addresses. For example, `127.0.0.1` will match requests to 127.0.0.1
                      - CIDR blocks. For example, `192.168.0.0./16` will match requests to any IP addresses in this range
                      - `*` will match all hosts
					
					If another no_proxy value is set in the configuration file or at a component level,
					this one will be overridden.
					
					The lowercase variant has priority over the uppercase one.
```